### PR TITLE
Add Python Chainguard Images tutorial

### PR DESCRIPTION
**Pull request by ltagliaferri**

### CODE REVIEW

File: content/chainguard/chainguard-images/reference/python/getting-started-python.md [hunk 1]
```diff
@@ -0,0 +1,259 @@
...
+
+```shell
+nano Dockerfile
+```
+
+The following Dockerfile will:
+
+1. Start a new build stage based on the `python:latest-dev` image and call it `builder`;
+2. Copy files from the current directory to the `/inky` location in the container;
+3. Enter the `/inky` directory and run `pip install -r requiements.txt` to install dependencies;
```
Comment (erikaheidi) [inline content/chainguard/chainguard-images/reference/python/getting-started-python.md:213]:
```suggestion
3. Enter the `/inky` directory and run `pip install -r requirements.txt` to install dependencies;
```

File: content/chainguard/chainguard-images/reference/python/getting-started-python.md [hunk 2]
```diff
@@ -0,0 +1,259 @@
...
+RUN pip install -r requirements.txt --user
+
+FROM cgr.dev/chainguard/python:latest
+
+WORKDIR /octo
+
+# Make sure you update Python version in path
+COPY --from=builder /home/nonroot/.local/lib/python3.11/site-packages /home/nonroot/.local/lib/python3.11/site-packages
+
+COPY wolfi.py chainguard.png .
```
Comment (erikaheidi) [inline content/chainguard/chainguard-images/reference/python/getting-started-python.md:236]:
```suggestion
COPY inky.py chainguard.png ./
```

File: content/chainguard/chainguard-images/reference/python/getting-started-python.md [hunk 3]
```diff
@@ -0,0 +1,259 @@
...
+
+ENTRYPOINT [ "python", "/inky/inky.py" ]
```
Comment (erikaheidi) [inline content/chainguard/chainguard-images/reference/python/getting-started-python.md:238]:
```suggestion
ENTRYPOINT [ "python", "/octo/inky.py" ]
```

**Review by erikaheidi — CHANGES_REQUESTED** [review]
I found a few path typos, other than that everything worked pretty well, and it is a really nice demo with inky!

File: content/chainguard/chainguard-images/reference/python/getting-started-python.md [hunk 4]
```diff
@@ -0,0 +1,259 @@
...
+mkdir ~/octo-facts/ && cd $_
+```
+
+Create a new file to serve as the application entry point. We’ll use `main.py`. You can edit this file in whatever code editor you would like. We'll use Nano as an example.
+
+```shell
+nano main.py
+```
+
+The following Python script defines a light CLI app that takes in a text file, `octo-facts.txt`, and returns a random lime from that file.
```
[the remark this answers is not in the document]
Comment (ltagliaferri) [reply content/chainguard/chainguard-images/reference/python/getting-started-python.md:54]: Haha, I would like a random lime 🍋

**Pull request merged**